pkg/asset/kubeconfig/kubelet.go

```diff
@@ -66,15 +66,15 @@ var _ asset.WritableAsset = (*KubeletClient)(nil)
 // Dependencies returns the dependency of the kubeconfig.
 func (k *KubeletClient) Dependencies() []asset.Asset {
 	return []asset.Asset{
-		&tls.KubeletClientCABundle{},
```

Comment:
Will the `kubelet-client-ca-bundle.crt` still be used by anything once it is removed from the kubelet kubeconfig?

Reply:
> Will the `kubelet-client-ca-bundle.crt` still be used by anything once it is removed from the kubelet kubeconfig?

I think it was wired improperly here before (kube-ca for everyone!).  It is logically distinct and is for verifying kubelet clients.